.github/workflows/tests_lkcpu_python.yml

```diff
@@ -244,8 +244,6 @@ jobs:
 
       - name: Run PennyLane-Lightning unit tests
         run: |
-          # TODO: Remove installing pytest-xdist with release v0.36.0
-          python -m pip install pytest-xdist
```

Comment:
Oops?

Reply:
Did we run py tests in parallel with `xdist` before?